integration-tests/smoke-tests/pom.xml

```diff
@@ -164,8 +164,21 @@
                                 <phase>compile</phase>
                             </execution>
                         </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <excludedGroups>development-only</excludedGroups>
+                            <reportNameSuffix>production</reportNameSuffix>
+                        </configuration>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-failsafe-plugin</artifactId>
                         <configuration>
-                            <productionMode>true</productionMode>
+                            <excludedGroups>development-only</excludedGroups>
```

Comment:
What is doing this line?

Reply:
The bootstrap test should run only in development mode. So it is annotated with a junit tag and skipped when running tests in production mode